### ISSUE TITLE
Parent id missing in metadata template scenario s370 

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#508]
I had again and again trouble with the time labels of the scenario run s370. 

I found one error in the metadata template file (`cmip6-output-control-files/ScenarioMIP/EC-EARTH-AOGCM/cmip6-experiment-ScenarioMIP-ssp370` from trunk):
```
     "parent_activity_id":           "",
     "parent_experiment_id":         "",
```
should be:
```
     "parent_activity_id":           "CMIP",
     "parent_experiment_id":         "historical",
```
Don't know yet if that solves all, or whether there is more going on. No idea yet why this could happen.

**Comment by treerink** [comment]
Correcting this seems to solve all the trouble. Haven't look into the cause of the error.

**Comment by treerink** [comment]
This is related to the `create-parent-experiment-check-code.py` which is used to construct the `modify-metadata-template.sh` script. I thought that an update of the `tables/CMIP6_CV.json` was the cause, but though its update causes a relevant update of the parent - index, it is not the cause of this issue. The issue here is that for some experiments in the `tables/CMIP6_CV.json` the `"activity_id"` field contains two MIPs, this is for the four scenarios only the case for `ssp3-7.0` Somehow, if possible `create-parent-experiment-check-code.py` needs to be made robust for this, as there are other cases with this issue.

**Comment by treerink** [comment]
From the `genecec` side this issue is solved in a521d1d. The updated metadata template files generated by `genecec` have to be brought to a new branch at the EC-Earth repository.

This fix (splitting the activity_id list when more than one MIP is listed in the CMIP6_CV.json) solves the `s370` case, but also more cases for other MIPs to come.

For here, closing the issue.
